pom.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0"?>
+<!--
+  ~ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License").
+  ~ You may not use this file except in compliance with the License.
+  ~ A copy of the License is located at
+  ~
+  ~  http://aws.amazon.com/apache2.0
+  ~
+  ~ or in the "license" file accompanying this file. This file is distributed
+  ~ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  ~ express or implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  -->
+
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.amazonaws</groupId>
+    <artifactId>log4j-CVE-2021-44228-hotpatch</artifactId>
+    <version>1.0</version>
```

Comment:
Should this be 1.0.0 (or even 1.0-SNAPSHOT to make it easier to pick up changes)?

Reply:
Either `1.0` or `1.0.0` should work, but Maven does not accept `SNAPSHOT` versions when publishing to maven.org

LMK if you'd prefer 1.0.0 and I can change it.

Reply:
In that case, I'd prefer `1.0.0`.

Reply:
Changed to `1.2` to match https://github.com/corretto/hotpatch-for-apache-log4j2/blob/0db8aafb3288e30e17040c80a57b04c0e20a3401/build.gradle#L9